ext/hal/nordic/nrfx/drivers/src/nrfx_usbd.c

```diff
@@ -265,7 +265,7 @@ static uint32_t m_ep_ready;
  * Mask prepared USBD data for transmission.
  * It is cleared when no more data to transmit left.
  */
-static uint32_t m_ep_dma_waiting;
+static atomic_t m_ep_dma_waiting;
```

Comment:
We use "nrfx_atomic_t" in nrfx, but it doesn't really matter here as nrfx comes from external source and it will have to be changed there.